static/index.js

```diff
@@ -95,7 +95,12 @@ class Flow {
 
       // 8. Create challenge iframe
       let creq = this.buildCReq(parsed);
-
+      // Convert https to http if url contains secure.localhost
+      //if (parsed.acsURL.includes('secure.localhost')) {
+      //#  console.log("Converting to http");
+      //parsed.acsURL = parsed.acsURL.replace('https', 'http');
+      //}
```

Comment:
Is this something we have used in other places? `localhost` is not a proper domain and using it this way seems quite smelly. 
We could add a `clearhaus`/`unzer` entry to peoples `hosts` instead but even that is quite annoying and most people don't link custom entries in their `hosts` file.

Reply:
Ooops some code.
Will remove it